test/infra/raft/cluster_config/one_version.pu_log_encrypt_symmetric.key

```diff
@@ -0,0 +1,5 @@
+keys {
```

Comment:
- [ ] is this a pesudo key?

Reply:
yes

Reply:
I feel it is not appropriate to include these encryption related test into Gringofts